### PR TITLE
feat(design-system): add single column and fixed header layouts showcases

### PR DESCRIPTION
**Pull request by mnavarroespinosa**
EX-7972

### CODE REVIEW

**Review by annacv — DISMISSED** [review]

File: packages/x-tailwindcss/demo/src/components/xds-layout-single-column.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            <div class="x-title2">HEADER MIDDLE</div>
+            <div class="x-title3">HEADER END</div>
+          </div>
+        </div>
+
+        <div class="x-layout-item x-border-b-1 x-border-neutral-25">
+          <div class="x-flex x-justify-center x-p-16 x-title3">SUB HEADER</div>
+        </div>
+
+        <div class="x-layout-item x-layout-expand x-layout-no-margin-right x-bg-neutral-25">
```
Comment (ajperezbau) [inline packages/x-tailwindcss/demo/src/components/xds-layout-single-column.vue:30]: I think here we can just add the `x-scroll` class to the item:

```
        <div
          class="x-layout-item x-scroll x-layout-expand x-layout-no-margin-right x-bg-neutral-25"
        >
          <div class="x-flex x-flex-col x-justify-between x-h-[2000px] x-p-12">
            <section>MAIN</section>
            <span class="x-place-self-end">MAIN BOTTOM</span>
          </div>
        </div>
```

**Review by ajperezbau — COMMENTED** [review]
Nice! 👌

File: packages/x-tailwindcss/demo/src/components/xds-layout-single-column.vue (hunk 1 above)
Comment (mnavarroespinosa) [reply packages/x-tailwindcss/demo/src/components/xds-layout-single-column.vue]: Good point! Finally, I have refactored it a bit.

**Review by ajperezbau — APPROVED** [review]

**Review by annacv — APPROVED** [review]
🚀

**Pull request merged**